support/cas-server-support-throttle-hazelcast/build.gradle

```diff
@@ -0,0 +1,36 @@
+description = "Apereo CAS Hazelcast Authentication Throttling"
+dependencies {
+    api project(":api:cas-server-core-api-throttle")
+
+    implementation libraries.hazelcast
+    implementation project(":support:cas-server-support-hazelcast")
+    implementation project(":support:cas-server-support-hazelcast-core")
+    implementation project(":core:cas-server-core-audit")
+    implementation project(":core:cas-server-core-audit-api")
+    implementation project(":support:cas-server-support-throttle")
+    implementation project(":support:cas-server-support-throttle-core")
+    implementation project(":core:cas-server-core-configuration-api")
+    implementation project(":core:cas-server-core-util-api")
+    
+
+    testImplementation project(path: ":support:cas-server-support-throttle", configuration: "tests")
+    testImplementation project(path: ":core:cas-server-core-authentication", configuration: "tests")
+    testImplementation project(path: ":core:cas-server-core-authentication-api", configuration: "tests")
+    testImplementation project(path: ":core:cas-server-core", configuration: "tests")
+    testImplementation project(path: ":core:cas-server-core-services", configuration: "tests")
+    testImplementation project(":core:cas-server-core-configuration")
+    testImplementation project(":support:cas-server-support-person-directory")
+    testImplementation project(":core:cas-server-core-web-api")
+    testImplementation project(":core:cas-server-core-web")
+    testImplementation project(":core:cas-server-core-tickets-api")
+    testImplementation project(":core:cas-server-core-tickets")
+    testImplementation project(":core:cas-server-core-services")
+    testImplementation project(path: ":core:cas-server-core")
```

Comment:
Path seems unnecessary here.

Also, it would be good to logically order the `tests` configurations separately from others. Easier on the eye.